# control : lateral



# control : longitudinal
float64 VMC_facAccnGvnrKpCurr_C	
float64 VMC_facAccnGvnrKpPosCurr_C	
float64 VMC_facAccnGvnrKpNegCurr_C	
float64 VMC_vAccnGvnrPWinPos_C	
float64 VMC_vAccnGvnrPWinNeg_C	
float64 VMC_facAccnGvnrKiCurr_C
float64 VMC_facAccnGvnrKiPosCurr_C
float64 VMC_facAccnGvnrKiNegCurr_C
float64 VMC_vAccnGvnrIWinPos_C
float64 VMC_vAccnGvnrIWinNeg_C
float64 VMC_swtAccnGvrnInil_C	
float64 VMC_aAccnGvnrOutpUp_C	
float64 VMC_aAccnGvnrOutpLo_C	
uint8  VMC_swtAccnGvnrFctEna_C	
float64 VMC_aAccnGvnrIntglCtrlValUp_C	
float64 VMC_aAccnGvnrIntglCtrlValLo_C	
float64 VDC_facRollngResCalCmp_C	
float64 VDC_moiDrvTrnCmp_C	
float64 EBS_aCmpFixd_C	
float64 CoChs_prcTrqHysLo_C	
float64 CoChs_prcTrqHysHi_C

